python3.html.markdown

```diff
@@ -145,6 +156,10 @@ bool({}) #=> False
 # Python has a print function
 print("I'm Python. Nice to meet you!")
 
+# By default the print function also prints out a newline at the end.
+# Use the optional argument end to change the end character.
+print("I'm Python. Nice to meet you!", end="")
```

Comment:
To show the power of `end`, could you make end be `*` or something then show what is executed by the code? 

example: `print("hello there", end="*") # => hello there*`